ReactWindows/ReactNative.Shared/Views/Text/FontStyleHelpers.cs

```diff
@@ -8,6 +9,23 @@ namespace ReactNative.Views.Text
 {
     static class FontStyleHelpers
     {
+        public static FontStyle? ParseFontStyle(string fontStyleString)
+        {
+#if WINDOWS_UWP
+            return EnumHelpers.ParseNullable<FontStyle>(fontStyleString);
+#else
+            if (fontStyleString == "normal")
+            {
+                return FontStyles.Normal;
+            }
+            else if (fontStyleString == "italic")
```

Comment:
the else is redundant since the previous clause is a return. here I would favor doing a switch statement (C# allows switch statements on strings):
```switch (fontStyleString)
{
  case "normal": return FontStyles.Normal;  
  case "italic": return FontStyles.Italic;
  default: return null;
}